integration/image/remove_test.go

```diff
@@ -58,3 +59,11 @@ func TestRemoveImageOrphaning(t *testing.T) {
 	_, _, err = client.ImageInspectWithRaw(ctx, commitResp2.ID)
 	testutil.ErrorContains(t, err, "No such image:")
 }
+
+func setupRegistry(t *testing.T) func() {
```

Comment:
where is this used?

Reply:
oh it was to port some registry tests to this one but shouldn't have commited that :sweat_smile: